mnemosyne/diagnose.py

```
@@ -266,6 +290,16 @@ def log(category: str, check: str, status: str, detail: str = ""):
         except Exception as exc:
             log("db", "memory_orphan_diagnostics", "ERROR", str(exc))
 
+        try:
+            from mnemosyne.core.hygiene import noise_summary as _noise_summary
+            hygiene = _noise_summary(Path(stats.get("database", "")), limit=200)
+            log("db", "hygiene_noise_scanned", str(hygiene.get("total_scanned", 0)))
+            log("db", "hygiene_noise_candidates", str(hygiene.get("total_candidates", 0)))
+            log("db", "hygiene_noise_ratio", str(hygiene.get("candidate_ratio", 0.0)))
+            log("db", "hygiene_noise_with_secrets", str(hygiene.get("with_secrets", 0)))
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)